src/main/kotlin/example/Header.kt

```diff
@@ -57,6 +59,32 @@ val Header = FC<Props> {
                 +"Kotlin MUI Showcase"
             }
 
+            Tooltip {
+                title = ReactNode(
+                    if (darkMode) {
+                        "Switch to light mode"
+                    } else {
+                        "Switch to dark mode"
+                    }
+                )
+
+                IconButton {
```

Comment:
Try to use `Switch` component. 
Seems to be the most acceptable solution for theming.